Message interviews:25 -  Read
From:      John Vlissides <vlis@interviews.stanford.edu>
To:        <slade.WBST128@Xerox.com>
Cc:        John Vlissides <vlis@lurch.stanford.edu>,
            InterViews Mailing List <interviews@interviews.stanford.edu>
Subject:   Re: MarginFrame
Obsoletes: Your message of Wed, 890802-132023-13727(a)Xerox

> I believe that there is a minor error in the fix17 patch as distributed-
>
> the section that attempts to patch the man page wants to use the file
>
>	"src/man/libInterViews/Frame.n"
>
> but in the distribution it is 	"man/libInterViews/Frame.n"

Oops! Here's the fix for the fix!

*** /user/vlis/fix17	Wed Aug  2 14:20:45 1989
--- fix17	Wed Aug  2 14:17:05 1989
***************
*** 8,14 ****

          src/InterViews/frame.h
          src/libInterViews/frame.c
!         src/man/libInterViews/Frame.n

  *** /tmp/da8373	Tue Aug  1 07:42:30 1989
  --- src/InterViews/frame.h	Tue Aug  1 07:33:26 1989
--- 8,14 ----

          src/InterViews/frame.h
          src/libInterViews/frame.c
!         man/libInterViews/Frame.n

  *** /tmp/da8373	Tue Aug  1 07:42:30 1989
  --- src/InterViews/frame.h	Tue Aug  1 07:33:26 1989
***************
*** 139,145 ****
  +
  + void MarginFrame::Redraw (Coord, Coord, Coord, Coord) { }
  *** /tmp/da8377	Tue Aug  1 07:43:16 1989
! --- src/man/libInterViews/Frame.n	Tue Aug  1 07:41:34 1989
  ***************
  *** 1,6 ****
    .TH Frame 3I "15 June 1987" "InterViews" "InterViews Reference Manual"
--- 139,145 ----
  +
  + void MarginFrame::Redraw (Coord, Coord, Coord, Coord) { }
  *** /tmp/da8377	Tue Aug  1 07:43:16 1989
! --- man/libInterViews/Frame.n	Tue Aug  1 07:41:34 1989
  ***************
  *** 1,6 ****
    .TH Frame 3I "15 June 1987" "InterViews" "InterViews Reference Manual"
